app/views/welcome_mailer/welcome_email.html.erb

```diff
@@ -0,0 +1,25 @@
+<h2 id="welcome-to-scholaruc">Welcome to <%= t('sufia.product_name') %></h2>
+<p>Welcome to <%= t('sufia.product_name') %>! This email summarizes helpful information for using <%= t('sufia.product_name') %>.</p>
+
+<h3 id="what-is-scholaruc">What is <%= t('sufia.product_name') %>?</h3>
+<p><%= t('sufia.product_name') %> is a digital repository that enables the University of Cincinnati community to share research and scholarly work with a worldwide audience and preserve it for the future. Read more about the mission of <%= t('sufia.product_name') %> on the <a href="/about">About page</a>. Get the latest updates on development of <%= t('sufia.product_name') %> on our <a href="http://libapps.libraries.uc.edu/scholarblog/">Scholarblog</a>. All users, including non-registered users, of <%= t('sufia.product_name') %> are bound by the <a href="/terms">Terms of Use</a>.</p>
```

Comment:
@13miciunasl Relative HTML links won't work because users will be clicking on them from their email client, not scholar.uc.edu.  So any relative links like `href="/about"` need to be absolute `href="https://scholar.uc.edu/about"` for this file and the student one.